src/frontend/styles/components/header.pcss

```diff
@@ -6,9 +6,14 @@
   border-bottom: 1px solid var(--color-line-gray);
   font-size: 18px;
   flex-wrap: wrap;
-  position: relative;
+  position: fixed;
+  top: 0;
+  box-sizing: border-box;
+  width: 100%;
+  z-index: 2;
+  background: white;
```

Comment:
why the bg is overridden?

Reply:
because bg is transparent which cause a issue